locale/en.po

```diff
@@ -3470,41 +3465,41 @@ msgstr ""
+"You LOSE health if you have no [thrive:compound type=\"atp\"][/thrive:compound]."
 
 msgid "MICROBE_STAGE_INITIAL"
 msgstr ""
-"On a distant alien planet, eons of volcanic activity and meteor impacts have led to the development of a new phenomenon in the universe.\n"
+"On a distant alien planet, eons of volcanic activity and meteor impacts have re-arranged matter into a remarkable new form.\n"
 "\n"
 "Life.\n"
 "\n"
-"Simple microbes reside in the deep regions of the ocean. You are the Last Universal Common Ancestor (LUCA) on this planet.\n"
+"This simple bubble has the ability to maintain a different environment inside of itself than in the water outside. In time, it may grow into a complex species that dominates the stars.\n"
 "\n"
-"To survive in this hostile world, you will need to collect any compounds that you can find and evolve across generations to compete against the other species of microbes."
+"But first, you will need to survive."
 
 msgid "MICROBE_STAGE_INITIAL_PANSPERMIA"
 msgstr ""
 "On a distant alien planet, a meteor has impacted after travelling vast astronomical distances carrying within it the seeds needed for a new phenomenon on the planet.\n"
 "\n"
 "Life.\n"
 "\n"
-"Simple microbes reside near the impact site. You are the Last Universal Common Ancestor (LUCA) on this planet.\n"
+"This microbe is generalized enough to survive in this ocean just as well as on that distant home world. Perhaps, one day, this microbe's distant decendants will find that world again.\n"
 "\n"
-"To survive in this hostile world, you will need to collect any compounds that you can find and evolve across generations to compete against the other species of microbes."
+"But first, you will need to survive."
 
 msgid "MICROBE_STAGE_INITIAL_POND"
 msgstr ""
-"On a distant alien planet, eons of chemical evolution building up to self-replicating molecules have led to the development of a new phenomenon in the universe.\n"
+"On a distant alien planet, eons of chemical evolution have built up to self-replicating molecules, and the development of a new phenomenon.\n"
 "\n"
 "Life.\n"
 "\n"
-"Simple microbes reside in shallow warm ponds. You are the Last Universal Common Ancestor (LUCA) on this planet.\n"
+"This simple bubble has the ability to maintain a different environment inside of itself than in the water outside. In time, it may grow into a complex species that dominates the stars.\n"
 "\n"
-"To survive in this hostile world, you will need to collect any compounds that you can find and evolve across generations to compete against the other species of microbes."
+"But first, you will need to survive."
 
 msgid "MICROBE_STAGE_ORGANELLE_DIVISION"
 msgstr ""
-"One of your organelles has divided.\n"
+"In the process of growing enough to reproduce, each of your organelles will divide. These new organelles provide bonuses (and osmoregulation costs) just like the organelles your microbe started with.\n"
```

Comment:
I take a bit of issue with calling new organelles providing bonuses, I'd cut that part to focus on that the new grown organelles act similarly to the ones the cell starts off with, there's just more of them.